code/game/objects/effects/decals/contraband.dm

```diff
@@ -145,9 +146,13 @@
 		if(ruined || !user.Adjacent(src))
 			return
 
+		if(user.a_intent == I_HELP)
+			user.examinate(src)
+			return
+
```

Comment:
The check to see if the person is on help intent should come before the alert().
Else they'll be asked if they want to rip the poster, and if they say yes and are on help intent nothing happens.